cartesian_impedance_msgs/msg/MaxCartesianVelocity.msg

```diff
@@ -0,0 +1,11 @@
+# parameters for Maximum allowed Cartesian Velocity when in Cartesian Impedance
```

Comment:
Can this message be replaced by a [Twist](http://docs.ros.org/api/geometry_msgs/html/msg/Twist.html) message